docs/Design_considerations.md

```diff
@@ -98,10 +124,26 @@ operations like multipart upload?
 ### Event logging 
 
 Log agent allowing log shipment to external system like kafka or ELK.
-Logs should be also reported to console or file?  
 We should decide on particular software library for this. For instance fluentd can be used to
 send messages to various sinks.
 
+#### logging infra components
+
+Apache Ranger requires Solr to store audit logs. DB audit is no longer supported. Solr is not preferred because
```

Comment:
Auditing is a must in Ranger how do you suggest solve it?

Reply:
If we stick with RangerDefaultAuditHandler mechanism from Ranger Authorizer, then we need to send logs to Solr. This is only option I guess. There is also kafkaProvider in agents-audit, but it is
not possible to set it as ResultProcessor.  At least not sure yet how to do it...
But it will only cover events generated by Ranger Audit. It we want to log some other events on s3proxy, then we will endup with another log reporter to kafka. Also we will not be able to customize audit logs generated by Ranger ResultProcessor. Eg. add requestId for further log corelation.
Maybe we should log audit events from Ranger using own log mechanism? To be discussed